subsys/bluetooth/host/classic/l2cap_br.c

```diff
@@ -32,75 +33,201 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_l2cap_br, CONFIG_BT_L2CAP_LOG_LEVEL);
 
-#define BR_CHAN_RTX(_w) CONTAINER_OF(k_work_delayable_from_work(_w), \
-				     struct bt_l2cap_br_chan, rtx_work)
+#if defined(CONFIG_BT_L2CAP_RET) || defined(CONFIG_BT_L2CAP_FC) ||                                 \
+	defined(CONFIG_BT_L2CAP_ENH_RET) || defined(CONFIG_BT_L2CAP_STREAM)
+#define L2CAP_BR_RET_FC_ENABLE 1
+#else
+#define L2CAP_BR_RET_FC_ENABLE 0
+#endif
+
+#define BR_CHAN_RTX(_w)                                                                            \
+	CONTAINER_OF(k_work_delayable_from_work(_w), struct bt_l2cap_br_chan, rtx_work)
+
+#define BR_CHAN_RET(_w)                                                                            \
+	CONTAINER_OF(k_work_delayable_from_work(_w), struct bt_l2cap_br_chan, ret_work)
+
+#define BR_CHAN_MONITOR(_w)                                                                        \
+	CONTAINER_OF(k_work_delayable_from_work(_w), struct bt_l2cap_br_chan, monitor_work)
+
+#define L2CAP_BR_PSM_START 0x0001
+#define L2CAP_BR_PSM_END   0xffff
+
+#define L2CAP_BR_CID_DYN_START    0x0040
+#define L2CAP_BR_CID_DYN_END      0xffff
+#define L2CAP_BR_CID_IS_DYN(_cid) (_cid >= L2CAP_BR_CID_DYN_START && _cid <= L2CAP_BR_CID_DYN_END)
+
+#define L2CAP_BR_MIN_MTU     48
+#define L2CAP_BR_DEFAULT_MTU 672
+
+#define L2CAP_BR_PSM_SDP 0x0001
+
+#define L2CAP_BR_S_FRAME_FLAG_MASK 0xFFFFFF00U
```

Comment:
Please decide if you want to use upper or lower case hex values. Now you're mixing both (lowercase on line 56 and upper case here). I've tried to use lower case everywhere in the Bluetooth host stack, but since there's so many contributors by now it's possible that there is some inconsistency. Please try to at least be consistent with yourself, or is there some reason why you're mixing these (e.g. special meaning with upper vs lower case?)

Reply:
> Please decide if you want to use upper or lower case hex values. Now you're mixing both (lowercase on line 56 and upper case here). 

Line 56 is changed due to clang-format. 

> I've tried to use lower case everywhere in the Bluetooth host stack,

I did not notice that. Let me update the Line 64.

Reply:
Updated all similar code lines.